patterns/1-initial/scoring-grid.md

```diff
@@ -0,0 +1,114 @@
+|         LICENSE        | No LICENSE file; obviously incorrect license info; e.g. default ISC for commercially sensitive repo; mismatch between license fields.                      | Appropriate LICENSE has been chosen; but the information is spread across multiple files e.g. at the headers of code files.                | Appropriate LICENSE file exists; and is consistently referenced in meta data such as README or package.json file.                       | Appropriate LICENSE file exists; additional information is provided, and thought has been given to the dependencies; there’s a way to verify the tree of license dependencies.                            |
+|     PULL REQUESTS      | No PULL REQUEST template; PR standards are defined externally to project; PR standards not applicable to this project.                                     | PULL REQUEST template exists; Advice for creating a PR exists in README or CONTRIBUTION files.                                             | PULL REQUEST template(s) exist for different types of changes. Main branch locked down. Named reviewers required.                       | PULL REQUEST standards are enforced by automated CI checks such as Danger.js; PR Templates contain checklist. CODEOWNERS lists required reviewers.                                                        |
+|       CODEOWNERS       | No CODEOWNERS file; ownership of repo unclear, or ownership belongs to a defunct team; contact details for team are outdated or incorrect.                 | CODEOWNERS file exists; named individuals required for review; main branch locked down; requires at least one person to provide PR review. | CODEOWNERS file exists; named team required for review; main branch locked down requiring PR review                                     | CODEOWNERS file exists; named team required for review, main branch locked down requiring PR review; accurate team contact details available in README.                                                   |
+| CONTINUOUS INTEGRATION | No Continuous Integration setup; builds are manual, and are transferred directly from developer laptop to production environment. Main branch fails tests. | Continuous Integration exists; requires manual triggers. Process not checked into source control. No PR integration.                       | Continuous Integration is well integrated; PRs automatically checked and verified. Process scripted as part of code base.               | Continuous Integration is fully automated; high trust test suite; features on main branch are automatically rolled out to production; use of canary builds; feature toggles.                              |
+
+Apply the grid against one or more repositories; deciding on a score. This can be done indiviually using an expert, or as team as a retrospective discussion.
```

Comment:
```suggestion
Apply the grid against one or more repositories; deciding on a score. This can be done individually using an expert, or as team as a retrospective discussion.
```

What do you mean by "individually using an expert"?